src/request_handler.cc

```diff
@@ -3,12 +3,13 @@
 #include <cstring>
 
 RequestHandler* RequestHandler::CreateByName(const char* type) {
-    // TODO(evan): use static registeration
-    std::string typeStr(type, std::strlen(type));
-    if (typeStr == "EchoHandler") {
-        return new EchoHandler();
-    } else {
-        std::cerr << "Unknown handler name '" << typeStr << "'" << std::endl;
+    if ( ! request_handler_builders) {
+        std::cerr << "No Request Handlers registered!" << std::endl;
```

Comment:
Hmmm yeah consistency would be good. I'll change this one and use `\n` in the future. I know I used `endl` in other places, but I'll leave those for now